beeline.go

```diff
@@ -118,44 +150,64 @@ func Close() {
 // an instrumented request. After adding the appropriate middleware or wrapping
 // a Handler, feel free to call AddField freely within your code. Pass it the
 // context from the request (`r.Context()`) and the key and value you wish to
-// add.
+// add.This function is good for span-level data, eg timers or the arguments to
+// a specific function call, etc. Fields added here are prefixed with `app.`
 func AddField(ctx context.Context, key string, val interface{}) {
-	ev := ContextEvent(ctx)
-	if ev == nil {
-		return
+	span := trace.GetSpanFromContext(ctx)
+	if span != nil {
```

Comment:
what about the cases where Events are created?  should we not also add the field to the event?

Reply:
Events aren't created as part of the core beeline API. So nope?

Reply:
going to chalk this up to confusion about wrappers creating both events and spans.

I haven't gone back and audited, but do all wrappers that send events send them from the method where they're created?  is that always going to be the case?